App.js

```diff
@@ -17,5 +44,118 @@ const styles = StyleSheet.create({
     backgroundColor: '#fff',
     alignItems: 'center',
     justifyContent: 'center',
+    fontFamily: 'Montserrat-Regular',
+  },
+  topBanner: {
+    flexDirection: 'row',
+    width: 400,
+    height: 300,
+    backgroundColor: '#FF715B',
+    position: 'absolute',
+    left: 0,
+    top: -2
+  },
+  loginText: {
+    fontSize: 80,
+    fontFamily: 'Montserrat-Medium',
+    fontWeight: '300',
+    color: '#fff',
+    position: 'absolute',
+    width: 347,
+    height: 170,
+    left: 13,
+    top: 115,
+    textAlign: 'center'
+  },
+  loginBox: {
+    flexDirection: 'row',
+    backgroundColor: '#fff',
+    position: 'absolute',
+    width: 347,
+    height: 370,
+    left: 14,
+    top: 250,
+    borderRadius: 25,
+    shadowOffset: {width: 0, height: 5},
+    shadowColor: '#000000',
+    shadowOpacity: 0.25,
+    shadowRadius: 5,
+  },
+  subText1: {
```

Comment:
`subText1` isn't descriptive enough - try to have something like `loginBox` or `loginText`. Also, don't include numbers in the name of the styles.